content/arduino-cloud/02.features/00.security-considerations/security-considerations.md

```diff
@@ -121,26 +121,31 @@ If it is determined that a breach has occurred, system(s) or system component(s)
 
 PII data will only be stored as long as Arduino legitimately needs it and in accordance with Arduino’s Privacy Policy.
 
-## Security Considerations for Specific Arduino Cloud Components
+## Security Considerations for Specific Arduino Cloud applications and tools
```

Comment:
```suggestion
## Arduino Cloud Applications and Tools
```

Reply:
I would shorten this heading, and seeing as we already are speaking about "security considerations" maybe we won't need it in the heading?

Reply:
I'm fine with the suggestion